Sources/QueryRef.swift

```diff
@@ -122,10 +122,10 @@ public protocol ObservableQueryRef: QueryRef {
   var lastError: DataConnectError? { get }
 }
 
-// QueryRef class used with ObservableObject protocol
-// data: Published variable that contains bindable results of the query.
-// lastError: Published variable that contains DataConnectError if last fetch had error.
-//            If last fetch was successful, this variable is cleared
+/// QueryRef class used with ObservableObject protocol
+/// data: Published variable that contains bindable results of the query.
+/// lastError: Published variable that contains DataConnectError if last fetch had error.
```

Comment:
The two parameters here don't match the type parameters. Also, symbols that should be links within the same module should be wrapped in triple backticks: 
```
```DataConnectError```
```

Reply:
They aren't type parameters. They are variables that this QueryRef publishes. 

I've updated docs for this and used the backticks to denote symbols. I've also documented the symbols within the class.